lib/rules/headers/h2-toc.js

```diff
@@ -5,18 +5,35 @@ exports.name  = 'headers.h2-toc';
 
 exports.check = function (sr, done) {
 
-    var EXPECTED_HEADING = /Table\ of\ Contents/;
-
-    var toc = sr.$('h2')
-    ,   found = false;
-
-    toc.each(function() {
-        found = found || EXPECTED_HEADING.test(sr.$(this).text());
-    });
-
-    if (!found) sr.warning(exports.name, 'missing');
-
-    return done();
+    var EXPECTED_HEADING = /table\s+of\s+contents/i;
+    var toc = sr.$('nav#toc > h2');
+
+    if (1 === toc.length) {
+        if (!EXPECTED_HEADING.test(sr.$(toc[0]).text())) {
+            sr.error(exports.name, 'not-found');
+        }
+    }
+    else {
+        toc = sr.$('#toc > h2');
```

Comment:
I thought so, too, @deniak.

To confirm, I checked the latest published specs ([example](https://www.w3.org/TR/2016/WD-webdriver-20160120/), [example](https://www.w3.org/TR/2016/WD-preload-20160119/), [example](https://www.w3.org/TR/2016/WD-vehicle-information-api-20160119/)) and saw that all of them are using `<section id="toc">`. Then I saw that [@fantasai is not enforcing one particular element in her CSS; only matching `#toc` (and she actually wrote `<section id="toc">` in the comments)](https://github.com/w3c/tr-design/blob/gh-pages/src/base.css#L9), so I thought we were fine with `section` too&hellip; That's why I'm matching only `#toc` here.

@fantasai, @plehegar: what needs an update? This Specberus rule, or the CSS file?